### PR TITLE
Update to latest vscodetestcover

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Update vscodetestcover with the new version that supports new 2015+ TS syntax. This allows us to remove the compiler version downgrade for azurecore and then re-enable the mssql tests. 

Note the esprima resolutions were necessary as a workaround for the issue described in the bug below - so removing those since they're no longer necessary either. 

Fixes https://github.com/microsoft/azuredatastudio/issues/9772

### CODE REVIEW

**Comment by aaomidi** [comment]
We should enable this and makes sure it still works https://github.com/microsoft/azuredatastudio/commit/1868a7d3705e4dc32b438ab24303052ed2cc9f09#r38379805

**Comment by Charles-Gagnon** [comment]
Huh, weird. Yeah re-enabling it to verify...

**Comment by Charles-Gagnon** [comment]
@aaomidi Ok so there's some weirdness going on with the mssql tests so I ended up re-disabling those. But this at least removes the compiler downgrade for azurecore so I'd like to get this in now and then work on getting the mssql tests fixed in a separate PR.

@anthonydresser Any comments on the disabling of the CI pipeline?

**Comment by Charles-Gagnon** [comment]
@aaomidi mssql tests aren't in because of the other issues which I'll follow up on - but this at least allows removing the compiler downgrade for the extension

@anthonydresser Any concerns over re-enabling the CI pipeline?

File: extensions/admin-tool-ext-win/package.json [hunk 1]
```diff
@@ -85,10 +85,7 @@
     "mocha-junit-reporter": "^1.17.0",
     "mocha-multi-reporters": "^1.1.7",
     "should": "^13.2.3",
-    "vscodetestcover": "github:corivera/vscodetestcover#1.0.5"
-  },
-  "resolutions": {
-      "esprima": "^4.0.0"
+    "vscodetestcover": "github:corivera/vscodetestcover#1.0.6"
```
Comment (aaomidi) [inline extensions/admin-tool-ext-win/package.json:88]: We should probably end up publishing this to npm?

**Pull request merged**